src/main/java/com/oltpbenchmark/api/collectors/DBCollector.java

```diff
@@ -59,5 +63,25 @@ public String collectVersion() {
         return version;
     }
 
+    public void collectDBParameters(Connection conn) {
+        try (Statement s = conn.createStatement()) {
 
+            // Collect DBMS version
+            try (ResultSet out = s.executeQuery("SELECT version();")) {
+                if (out.next()) {
+                    this.version = out.getString(1);
+                }
+            }
+
+            // Collect DBMS parameters
+            try (ResultSet out = s.executeQuery("SHOW ALL;")) {
```

Comment:
Convert back to `final strings` for the statements.
Also, have we tested that those work for all DBs?